packages/jest-cli/src/__tests__/TestRunner-test.js

```diff
@@ -38,6 +38,115 @@ test('.addReporter() .removeReporter()', () => {
   expect(runner._dispatcher._reporters).not.toContain(reporter);
 });
 
+describe('noDefaultReporters option', () => {
```

Comment:
Adding test for noDefaltReporters option. As I am using this same option for testing the reporters in the next describe function block too.